### PR TITLE
doc/rados/operations: edit monitoring.rst

### PR DESCRIPTION
**Pull request by zdover23**
Line-edit the final third of doc/rados/operations/monitoring.rst.

Follows https://github.com/ceph/ceph/pull/50834.

https://tracker.ceph.com/issues/58485

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/rados/operations/monitoring.rst [hunk 1]
```diff
@@ -404,48 +404,49 @@ The output of ``ceph df`` resembles the following::
...
+  BlueStore's database).
 - **RAW USED:** The amount of raw storage consumed by user data, internal
-  overhead, or reserved capacity.
+  overhead, and reserved capacity.
 - **%RAW USED:** The percentage of raw storage used. Use this number in
-  conjunction with the ``full ratio`` and ``near full ratio`` to ensure that 
-  you are not reaching your cluster's capacity. See `Storage Capacity`_ for 
-  additional details.
+  conjunction with ``full ratio`` and ``near full ratio`` to ensure that 
+  you are not reaching your cluster's capacity. See `Storage Capacity`_.
```
Comment (anthonyeleven) [inline doc/rados/operations/monitoring.rst:417]: Reaching sounds like failing to be all that we can be, in a way.
Suggest something like "Watch this number in conjunction with ... to be forewarned if your cluster approaches the fullness thresholds."

File: doc/rados/operations/monitoring.rst [hunk 2]
```diff
@@ -404,48 +404,49 @@ The output of ``ceph df`` resembles the following::
...
 
-- **ID:** The number of the node within the pool.
-- **STORED:** actual amount of data user/Ceph has stored in a pool. This is
-  similar to the USED column in earlier versions of Ceph but the calculations
-  (for BlueStore!) are more precise (gaps are properly handled).
+- **ID:** The number of the specific node within the pool.
+- **STORED:** The actual amount of data that the user has stored in a pool.
+  This is similar to the USED column in earlier versions of Ceph, but the
+  calculations (for BlueStore!) are more precise (in that gaps are properly
+  handled).
```
Comment (anthonyeleven) [inline doc/rados/operations/monitoring.rst:432]: Suggest removing the second set of parens, and the first parenthetical entirely (for Reef and later!)

**Review by anthonyeleven — APPROVED** [review]
suggestions

File: doc/rados/operations/monitoring.rst [hunk 3]
```diff
@@ -404,48 +404,49 @@ The output of ``ceph df`` resembles the following::
    cephfs.a.meta           2   32  6.8 KiB  6.8 KiB      0 B        22   96 KiB  96 KiB      0 B       0    297 GiB            N/A          N/A     22         0 B          0 B
    cephfs.a.data           3   32      0 B      0 B      0 B         0      0 B     0 B      0 B       0     99 GiB            N/A          N/A      0         0 B          0 B
    test                    4   32   22 MiB   22 MiB   50 KiB       248   19 MiB  19 MiB   50 KiB       0    297 GiB            N/A          N/A    248         0 B          0 B
-
-- **CLASS:** for example, "ssd" or "hdd"
+   
+- **CLASS:** For example, "ssd" or "hdd".
```
[the remark this answers is not in the document]
Comment (zdover23) [reply doc/rados/operations/monitoring.rst:408]: unfinished business -- use/mention standard decision to be taken later in 2023

File: doc/rados/operations/monitoring.rst (hunk 2 above)
Comment (zdover23) [reply doc/rados/operations/monitoring.rst:432]: unfinished business

**Pull request merged**